### PR TITLE
Fixed php 7.3 compatibility

### PR DESCRIPTION
**Pull request by alexander-schranz**

### CODE REVIEW

File: Result/Converter.php [hunk 1]
```diff
@@ -94,7 +94,7 @@ public function assignArrayToObject(array $array, $object, array $aliases)
                 switch ($aliases[$name]['type']) {
                     case 'date':
                         if (is_null($value) || (is_object($value) && $value instanceof \DateTimeInterface)) {
-                            continue;
```
[the remark this answers is not in the document]
Comment (saimaz) [reply Result/Converter.php]: yes

[the remark this answers is not in the document]
Comment (alexander-schranz) [reply Result/Converter.php]: @saimaz on older php versions there is no difference between continue; and continue 2;
tested with a small script:

```
php -r 'foreach ([1, 2] as $test) {
  echo $test;
  switch($test) {
      default:
          continue 2;
    }
}'
```

[the remark this answers is not in the document]
Comment (saimaz) [reply Result/Converter.php]: Thank you for the double check.

**Comment by alexander-schranz** [comment]
@saimaz seems like there is still issues with elasticsearch php adapter: https://github.com/elastic/elasticsearch-php/pull/803

**Pull request merged**